src/header/css/header-menus.pcss

```diff
@@ -446,7 +446,23 @@
 				&[data-structuredcols=""] {
 					width: calc(300px * 2 + 20px + 35px * 2);
 					&[data-hasMultiCols="true"] {
-						width: calc(290px * 3 + 20px + 20px * 3);
+						&[data-filled="12"],
```

Comment:
this is getting a little wacky. I would maybe just pass a modifier css class to these columns from JS?

Reply:
@paulcpederson updated this logic.. should be cleaner.